### PR TITLE
Cleanup OIDC Config Mirage handler

### PR DESCRIPTION
**Pull request by hellobontempo**

### CODE REVIEW

File: ui/tests/acceptance/oidc-config/clients-test.js [hunk 1]
```diff
@@ -41,7 +41,16 @@ module('Acceptance | oidc-config/clients', function (hooks) {
 
   test('it renders empty state when no clients are configured', async function (assert) {
     assert.expect(5);
-    this.server.get('/identity/oidc/client', () => overrideMirageResponse(200));
+
+    // * ENSURE CLEAN TEST STATE
+    try {
+      const model = await this.store.findRecord('oidc/client', 'some-app');
+      model.destroyRecord();
```
Comment (Monkeychip) [inline ui/tests/acceptance/oidc-config/clients-test.js:48]: Reading the Ember API docs on [destroyRecord](https://api.emberjs.com/ember-data/release/classes/Model/methods/destroyRecord?anchor=destroyRecord) it looks like it returns a promise, so maybe put an await in front of this as well and then you wouldn't need the settled? Not 100% sure, but just a thought.

File: ui/tests/acceptance/oidc-config/clients-test.js [hunk 2]
```diff
@@ -58,6 +67,16 @@ module('Acceptance | oidc-config/clients', function (hooks) {
 
   test('it creates, updates and deletes a client', async function (assert) {
     assert.expect(20);
+
+    // * ENSURE CLEAN TEST STATE
+    try {
+      const model = await this.store.findRecord('oidc/client', 'some-app');
+      model.destroyRecord();
```
Comment (Monkeychip) [inline ui/tests/acceptance/oidc-config/clients-test.js:74]: same here.

**Review by Monkeychip — APPROVED** [review]
Nothing blocking. Just a comment.

File: ui/tests/acceptance/oidc-config/clients-test.js (hunk 1 above)
Comment (hellobontempo) [reply ui/tests/acceptance/oidc-config/clients-test.js:48]: ah - you know. i think using a `.then` here would be cleanest and ensure the promises chain and are handled correctly!

**Pull request merged**